client/state/stats/lists/selectors.js

```diff
@@ -201,9 +201,22 @@ export function getSiteStatsViewSummary( state, siteId ) {
 		viewSummary[ years ][ months ].data.push( item );
 		const average =
 			viewSummary[ years ][ months ].total / viewSummary[ years ][ months ].daysInMonth;
-		viewSummary[ years ][ months ].average = round( average, 0 );
+		viewSummary[ years ][ months ].average = Math.round( average );
 	} );
 
+	// With the current month, calculate the average based on the days passed so far in the month
+	if ( viewData.data.length ) {
+		const momentTime = moment();
```

Comment:
I've taken another look at this and I think the `moment` usage here is really unnecessary - it could be replaced with simple `Date()` usage:

* Instead of `moment()` , we can use `new Date()`.
* Instead of `momentTime.year()` we can use `new Date().getFullYear()`
* Instead of `momentTime.month()` we can use `new Date().getMonth() + 1` (because months are zero-based)
* Instead of `const daysSinceStartOfMonth = moment( new Date() ).diff( momentTime.startOf( 'month' ), 'days' ) + 1;` we can use `new Date().getDate()`.

That being said, I'd love it if we can avoid introducing `moment` in the tests, too.

Reply:
Thank you! I've now removed the usage of `moment`, including in the tests. :)